.github/PULL_REQUEST_TEMPLATE/production.md

```diff
@@ -21,7 +21,7 @@ please add links to any relevant follow up issues.*
 I have...
 
 * [ ] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-* [ ] Added `!` to the type prefix if API or client breaking change
+* [ ] Added `!` to the type prefix if state-machine breaking change (i.e., requires coordinated upgrade)
```

Comment:
Why not have all 3 items listed under ! features?
API, Client & State Breaking.